src/api/models/sql_models/benthicpqt.py

```diff
@@ -0,0 +1,309 @@
+                    JOIN se ON su.sample_event_id = se.sample_event_id
+                GROUP BY
+                    tt_1.quadrat_transect_id
+            ) observers ON su.id = observers.quadrat_transect_id
+            JOIN (
+                WITH RECURSIVE tree(child, root) AS (
+                    SELECT
+                        c_1.id,
+                        c_1.id
+                    FROM
+                        benthic_attribute c_1
+                    LEFT JOIN benthic_attribute p ON c_1.parent_id = p.id
+                    WHERE
+                        p.id IS NULL
+                    UNION
+                    SELECT
+                        benthic_attribute.id,
+                        tree_1.root
+                    FROM
+                        tree tree_1
+                        JOIN benthic_attribute ON tree_1.child = benthic_attribute.parent_id
+                )
+                SELECT
+                    tree.child,
+                    tree.root
+                FROM
+                    tree
+            ) category ON o.attribute_id = category.child
+            JOIN benthic_attribute cat ON category.root = cat.id
+            JOIN benthic_attribute b ON o.attribute_id = b.id
+            LEFT JOIN growth_form gf ON o.growth_form_id = gf.id
+            LEFT JOIN api_current c ON su.current_id = c.id
+            LEFT JOIN api_tide t ON su.tide_id = t.id
+            LEFT JOIN api_visibility v ON su.visibility_id = v.id
+            LEFT JOIN api_relativedepth r ON su.relative_depth_id = r.id
+            LEFT JOIN api_reefslope rs ON su.reef_slope_id = rs.id
+    """
+
+    sql_args = dict(project_id=SQLTableArg(required=True))
+
+    objects = SQLTableManager()
+
+    sample_unit_id = models.UUIDField()
+    transect_number = models.PositiveSmallIntegerField()
+    relative_depth = models.CharField(max_length=50)
+    transect_len_surveyed = models.PositiveSmallIntegerField(
+        verbose_name=_("transect length surveyed (m)")
+    )
+    quadrat_size = models.DecimalField(decimal_places=2, max_digits=6)
+    num_quadrats = models.PositiveSmallIntegerField()
+    num_points_per_quadrat = models.PositiveSmallIntegerField()
+    reef_slope = models.CharField(max_length=50)
+    benthic_category = models.CharField(max_length=100)
+    benthic_attribute = models.CharField(max_length=100)
+    growth_form = models.CharField(max_length=100)
+    num_points = models.PositiveSmallIntegerField()
+    observation_notes = models.TextField(blank=True)
+    data_policy_benthicpqt = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = "benthicpqt_obs_sm"
+        managed = False
+
+
+class BenthicPhotoQuadratTransectSUSQLModel(BaseSUSQLModel):
+    # Unique combination of these fields defines a single (pseudo) sample unit. All other fields are aggregated.
+
+    # TODO: Does anything specific to Photo quadrat should be defined in this list
```

Comment:
Discussed with Emily; I think we can leave this as is for now (and remove comment).